CMakeLists.txt

```diff
@@ -128,9 +128,6 @@ if (CMAKE_CXX_COMPILER_ID MATCHES ".*Clang"
   # PREPEND the options.
   set(CMAKE_CXX_FLAGS "${H2_CMAKE_CXX_FLAGS_STR} ${CMAKE_CXX_FLAGS}")
 
-  # Disable all optimization in debug.
-  list(APPEND CMAKE_CXX_FLAGS_DEBUG -O0)
```

Comment:
@benson31  Just to make sure, did you remove the lines on purpose?

Reply:
Yes, they're not needed.